src/ConventionalCommitUtilties.js

```diff
@@ -0,0 +1,61 @@
+import ChildProcessUtilities from "./ChildProcessUtilities";
+import FileSystemUtilities from "./FileSystemUtilities";
+import logger from "./logger";
+import semver from "semver";
+
+const CHANGELOG_HEADER = `# Change Log
+
+All notable changes to this project will be documented in this file.
+See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.`;
+
+export default class ConventionalCommitUtilties {
+  @logger.logifySync()
+  static recommendVersion(name, version) {
+    try {
+      const pkgPath = `./packages/${name}/`;
```

Comment:
@evocateur yeah, I noticed this when I started writing tests; wasn't sure how complex `pkg.location` could become.

It feels like `changelog.location` should then be something like `path.resolve(pkg.location, 'changelog.md')`?